src/legend.ts

```diff
@@ -106,9 +123,18 @@ export interface LegendConfig {
    */
   titleFontWeight?: string | number;
   /**
-   * Optional mark definitions for custom legend encoding.
+   * The padding, in pixels, between title and legend.
    */
-  encode?: VgLegendEncode;
+  titlePadding?: number;
+  // ---------- Other ----------
+  /**
+   * Padding (in pixels) between legend entries in a symbol legend.
+   */
+  entryPadding?: number;
```

Comment:
`entryPadding` and `tickCount` seem to be out of place here. 

For general Legend properties, put them on top.  Within each category, sort alphabetically.